TRAMS-API/Models/Upstream/Response/GetProjectsResponseModel.cs

```diff
@@ -0,0 +1,139 @@
+
+    public class GetProjectsAcademyResponseModel
+    {
+        /// <summary>
+        /// The ID of the project's academy
+        /// </summary>
+        public Guid ProjectAcademyId { get; set; }
+
+        /// <summary>
+        /// The ID of the parent Academy Transfers project
+        /// </summary>
+        public Guid ProjectId { get; set; }
+
+        /// <summary>
+        /// The TRAMS ID of the academy
+        /// </summary>
+        public Guid AcademyId { get; set; }
+
+        /// <summary>
+        /// The name of the academy
+        /// </summary>
+        /// <example>Imaginary Academy</example>
+        public string AcademyName { get; set; }
+
+        /// <summary>
+        /// The ESFA intervention reasons selection for this project's academy
+        /// </summary>
+        public List<EsfaInterventionReasonEnum> EsfaInterventionReasons { get; set; }
+
+        /// <summary>
+        /// An explanation of the ESFA intervention reasons
+        /// </summary>
+        /// <example>The ESFA Intervention Reasons explained</example>
+        public string EsfaInterventionReasonsExplained { get; set; }
+
+        /// <summary>
+        /// The RDD or RSC Intervention reasons
+        /// </summary>
+        public List<RddOrRscInterventionReasonEnum> RddOrRscInterventionReasons { get; set; }
+
+        /// <summary>
+        /// An exaplanation of the RDD or RSC Intervention reasons
+        /// </summary>
+        /// <example>The RDD or RSC Intervention reasons explained</example>
+        public string RddOrRscInterventionReasonsExplained { get; set; }
+
+        /// <summary>
+        /// Information about the proposed incoming trusts for this academy
+        /// Data only returned on requesting the project's academy by id
+        /// </summary>
+        public List<GetAcademyTrustsResponseModel> AcademyTrusts { get; set; }
+    }
+
+    public class GetProjectsTrustResponseModel
+    {
+        /// <summary>
+        /// The ID of the project's identified incoming trust
+        /// </summary>
+        public Guid ProjectTrustId { get; set; }
+
+        /// <summary>
+        /// The TRAMS ID of the trust
+        /// </summary>
+        public Guid TrustId { get; set; }
+
+        /// <summary>
+        /// The name of the trust
+        /// </summary>
+        /// <example>Imaginary Trust</example>
+        public string TrustName { get; set; }
+    }
+
+    public class GetAcademyTrustsResponseModel
+    {
+        /// <summary>
+        /// The ID of the project's identified incoming trust
+        /// </summary>
+        public Guid ProjectTrustId { get; set; }
+
+        /// <summary>
+        /// The TRAMS ID of the trust
+        /// </summary>
+        public Guid TrustId { get; set; }
+
+        /// <summary>
+        /// The name of the trust
+        /// </summary>
+        /// <example>Imaginary Trust</example>
+        public string TrustName { get; set; }
+
+        public int Priority { get; set; }
```

Comment:
Suggestion: Make this an enum or add some guidance around expected values.

Reply:
Will fix